src/plots/cartesian/graph_interact.js

```diff
@@ -17,8 +17,9 @@ var Plotly = require('../../plotly');
 var Lib = require('../../lib');
 var Events = require('../../lib/events');
 
-var prepSelect = require('./select');
 var constants = require('./constants');
+var dragBox = require('./dragbox');
+var dragElement = require('../../components/dragElement');
```

Comment:
:facepunch: dammit autocomplete! Thanks. Weird that it works here, not like I'm on Windoze where everyone is used to case-insensitive filenames.